code/datums/redis/callbacks/_redis_callback.dm

```diff
@@ -19,7 +19,7 @@
  * * message - The message received on the redis channel.
  */
 /datum/redis_callback/proc/on_message(message)
-	CRASH("on_message not overriden for [type]!")
+	CRASH("on_message not overridden for [type]!")
```

Comment:
This change may be notable if specific errors are being tracked across multiple rounds in some kind of database, if so, I can change this back.

Reply:
This is fine.